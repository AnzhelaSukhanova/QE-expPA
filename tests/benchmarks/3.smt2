(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (forall ((x (_ BitVec 4))) (and 
 (or (bvult (bvmul y #b0011) x) (= (bvmul y #b0011) x)) 
 (or (bvult x (bvmul y #b1100)) (= x (bvmul y #b1100))))))
