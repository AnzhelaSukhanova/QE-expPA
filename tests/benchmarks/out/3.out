(and (= (bvmul y #xc) #xf) (= (bvmul y #x3) #x0))
